client/views/actions/topic_set_by.tpl

```diff
@@ -0,0 +1 @@
+Topic set by <a href="#" class="user" data-name="{{nick}}">{{mode}}{{nick}}</a> on {{localeDate when}}
```

Comment:
Can we combine `topic` and `topic_set_by` templates? It would make future changes/additions more self-contained, and also a nicer experience for the user.

Right now:
- if someone changes the topic, we get:

  ```
<nick> has changed the topic to: <topic>
```
- When joining a channel:

  ```
The topic is: <topic>
Topic set by <nick> on <date>
```

I'd see something more "uniform" in both cases:

```
Topic set[ by <nick>][ on <date>] to: <topic>
```

If it is possible to not know what `nick` set the topic, do not display `by <nick>` and, when nick is changed, do not display `on <date>` (although I'd be even in favor of not displaying the date altogether but without a strong opinion).

What do you think?